packages/q/qt5base/xmake.lua

```diff
@@ -0,0 +1,220 @@
+local function qt_table(sdkdir, version)
+    return {
+        version = version,
+        sdkdir = sdkdir,
+        sdkver = version,
+        bindir = path.join(sdkdir, "bin"),
+        includedir = path.join(sdkdir, "include"),
+        libdir = path.join(sdkdir, "lib"),
+        libexecdir = path.join(sdkdir, "libexec"),
+        mkspecsdir = path.join(sdkdir, "mkspecs"),
+        qmldir = path.join(sdkdir, "qml"),
+        pluginsdir = path.join(sdkdir, "plugins")
+    }
+end
```

Comment:
Why not use qmake query to get them, it should be more accurate. https://github.com/xmake-io/xmake/blob/bcf29f16d1ffd65b07a11e80e08ebac42eef4083/xmake/modules/detect/sdks/find_qt.lua#L162-L176

Reply:
Because it won't work for cross-platform development (MinGW on macosx, maybe cross/ARM)

Reply:
ok